compliance-web/src/components/App/Inspections/Profile/InspectionEnforcements.tsx

```diff
@@ -258,18 +276,18 @@ const InspectionEnforcements: React.FC<InspectionEnforcementsProps> = ({
           />
         );
         break;
-      case EnforcementActionEnum.CHARGE_RECOMMENDATION:
-        content = (
-          <ChargeRecommendationCreateModal
-            inspectionData={inspectionData}
-            requirementsList={nonProceededChargeRecommendationRequirements}
-            requirement={requirement}
-            onSubmit={() => {
-              setModalClose();
-            }}
-          />
-        );
-        break;
+             case EnforcementActionEnum.CHARGE_RECOMMENDATION:
+         content = (
+           <ChargeRecommendationCreateModal
+             inspectionData={inspectionData}
```

Comment:
Please test this because there might be backend changes in relation to this. In the create and update methods of ChargeRecommendations, I am checking whether the requirement has a charge recommendation mapped to the requirement or not which we don't have to check